src/Config/ConfigInterface.php

```diff
@@ -52,6 +52,13 @@ public function getEntryPoints(): array;
      */
     public function getAssetFiles(): array;
 
+    /**
+     * Return a list of files to exclude when bundeling. These are the files defined under 'excludes'.
+     *
+     * @return string[]
+     */
+    public function getExcludeImports(): array;
```

Comment:
`getExcludedImports`
`getProvidedImports`
`getIgnoredImports`

I like `getExcludedImports` the best from that list, but not 100% sure about the name yet, though.